MonMaraicher/MapView.swift

```diff
@@ -10,9 +10,32 @@ import MapKit
 
 struct MapView: View {
     
+    @StateObject private var viewModel = MapViewModel()
+    
     var body: some View {
-        
-        Text("Hello world!")
+        Map(position: $viewModel.userPosition, selection: $viewModel.selectedFarmerPlace) {
+            
+            ForEach(viewModel.allFarmerPlaces, id: \.id) { place in
+                Marker(place.name, coordinate: CLLocationCoordinate2D(latitude: place.location.latitude, longitude: place.location.longitude))
+                    .tag(place)
+                    .tint(.green)
+            }
+            
+            UserAnnotation()
+            
+        }
+        .onChange(of: viewModel.selectedFarmerPlace, { oldValue, newValue in
+            viewModel.showDetails = newValue != nil
+        })
+        .sheet(isPresented: $viewModel.showDetails, content: {
+            DetailFarmerView(selectedFarmer: viewModel.selectedFarmerPlace, show: $viewModel.showDetails)
+                .presentationDetents([.height(200), .medium, .large])
```

Comment:
Why 200px? Magic number? 😅

Reply:
It was a test. This value no longer exists